File: src/org/funivan/intellij/FastCoddy/CodeBuilders/CodeBuilderInterface.java
package org.funivan.intellij.FastCoddy.CodeBuilders;

import com.intellij.psi.PsiFile;

import org.jetbrains.annotations.Nullable;

/**
 * @author Ivan Scherbak <dev@funivan>
 */
interface CodeBuilderInterface {

    CodeTemplate expandCodeFromShortcut(String shortcut, PsiFile psiFile);


}

File: src/main/kotlin/org/funivan/intellij/FastCoddy/CodeBuilders/CodeBuilderInterface.kt
package org.funivan.intellij.FastCoddy.CodeBuilders;

import com.intellij.psi.PsiFile;

import org.jetbrains.annotations.Nullable;

/**
 * @author Ivan Shcherbak <alotofall@gmail.com>
 */
interface CodeBuilderInterface {

    CodeTemplate expandCodeFromShortcut(String shortcut, PsiFile psiFile);


}
